src/api/personal.rs

```diff
@@ -151,4 +163,10 @@ mod tests {
       ::serde_json::from_str(EXAMPLE_TX).unwrap()
       => ::serde_json::from_str::<RawTransaction>(EXAMPLE_TX).unwrap()
     );
+
+    rpc_test! {
+      Personal:import_raw_key, "0000000000000000000000000000000000000000000000000000000000000123", "hunter2" =>
```

Comment:
What the test is doing is calling `Personal::import_raw_key` with the two parameters you have here, next line checks that it's properly serialized as JSON-RPC request and the last line is JSON-RPC response and how it's supposed to be deserialized. So it looks good then :)

Reply:
Great; thank you!